cell_type_prediction/README.md

```diff
@@ -56,5 +56,49 @@ Two output tables are produced, the first containing a list of all predicted lab
 4) Semantic similarity across predicted labels. Correctly predicted cells are likely to have labels that are highly similar among them (think high precision). 
 5) Aggregated score: again, scores  2-4 are averaged to produce combined score. Cells are sorted by this score. Those cells that have low score are less likely to be correctly predicted. 
 
+## Use case: run cell type predictions programmatically
+It is possible to run the workflow programmatically using the [galaxy-workflow-executor](https://github.com/ebi-gene-expression-group/galaxy-workflow-executor) package. For more detail, see the README on the provided repository. You will need [conda](https://docs.conda.io/en/latest/) to install it. Run the following commands: 
+
+```
+conda create -n <env_name> galaxy-workflow-executor
+conda activate <env_name>
+```
+
+The following steps will walk you through the procedure. Example data is provided in [galaxy_cli_use_case](galaxy_cli_use_case) directory.
```

Comment:
Maybe clarify here that this is done only once per workflow file only (not per data execution).